SR2025/2024-12-18-check-in.md

```diff
@@ -0,0 +1,20 @@
+---
+to: SR2025 teams
+subject: Student Robotics 2025 {TLA} December Check-in
+---
+
+Hi {PrimaryGivenName},
+
+Just wanted to check in on how your team is doing?
+
+It would be great to hear what your team is currently working on and if there's
+anything you'd like a hand with.
+
+As a reminder, if you'd like to come to the [Cambridge Tech Day][cambridge-tech-day]
+please [let us know][tech-day-signup] this week.
+
+Thanks,\
```

Comment:
The trailing `\` here ensures this is a line break, rather than being unwrapped or becoming a paragraph break.